tests/integration/before-you-start-site-details/before-you-start-site-details.test.js

```diff
@@ -0,0 +1,152 @@
+import { getByRole, getByText } from '@testing-library/dom'
+import { JSDOM } from 'jsdom'
+import { routes } from '~/src/server/common/constants/routes.js'
+import { statusCodes } from '~/src/server/common/constants/status-codes.js'
+import { getExemptionCache } from '~/src/server/common/helpers/session-cache/utils.js'
+import { createServer } from '~/src/server/index.js'
+
+jest.mock('~/src/server/common/helpers/session-cache/utils.js')
+
+describe('Before you start site details page', () => {
+  let server
+
+  const mockExemption = {
+    id: 'test-exemption-123',
+    projectName: 'Test Project'
+  }
+
+  beforeAll(async () => {
+    server = await createServer()
+    await server.initialize()
+  })
+
+  afterAll(async () => {
+    await server.stop()
+  })
+
+  beforeEach(() => {
+    jest.resetAllMocks()
```

Comment:
We could probably put this once in jest setup to save having to remember to add it to each test file?